frontend/public/components/dashboard/dashboards-page/cluster-dashboard/status-card.tsx

```diff
@@ -98,8 +98,8 @@ const cvResource: WatchK8sResource = {
   isList: false,
 };
 
-const ClusterAlerts = withDashboardResources(
-  ({ watchAlerts, stopWatchAlerts, notificationAlerts }) => {
+export const DashboardAlerts = withDashboardResources<DashboardItemProps & DashboardAlertsProps>(
+  ({ watchAlerts, stopWatchAlerts, notificationAlerts, labelFilterKey, labelFilterValue }) => {
```

Comment:
A better API is to take in an object of key/value pairs for the label filter. Then you can filter on multiple labels if you need to, and it better matches what the k8s API itself uses.

Reply:
Added the object of key/value pairs for the label filter